lib/msal-angular/src/msal.guard.ts

```diff
@@ -129,8 +129,11 @@ export class MsalGuard implements CanActivate, CanActivateChild, CanLoad {
         // Capture current path before it gets changed by handleRedirectObservable
         const currentPath = this.location.path(true);
 
-        return this.authService.handleRedirectObservable()
+        return this.authService.initialize()
```

Comment:
Same question here. Why are we replacing `handleRedirectObservable()` with `initialize()`?

Reply:
We're not replacing, we're just calling `initialize()` before `handleRedirectObservable()`. This is because `allowNativeBroker` is set to true by default in v3, and we need to account for [this error](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/errors.md#native_broker_called_before_initialize). This is already being done in the MsalRedirectComponent, but needed to be updated here.

Reply:
Gotcha, thanks!